testing/scenario_app/ios/Scenarios/ScenariosTests/AppLifecycleTests.m

```diff
@@ -122,7 +122,44 @@ - (void)testLifecycleChannel {
       addObject:[[XCTestExpectation alloc]
```

Comment:
I expect that this test will fail earlier once @gaaclarke's channel buffer comes through right? Right now it's not crashing because of an extraneous detached signal at the beginning because the detached message you're sending on the FlutterEngine construct is just being dropped on the floor because no one is listening by that point on the framework right?

Reply:
yes